pypeman/channels.py

```diff
@@ -480,6 +495,51 @@ def graph_dot(self, end=''):
             for entry in sub.graph_dot(end=end):
                 yield entry
 
+    def _init_callback_nodes(self, *callback_nodes):
+        callback_nodes = [node for node in flatten(callback_nodes) if node]
+        for node in callback_nodes:
+            node.channel = self
+
+        if len(callback_nodes) > 1:
+            previous_node = callback_nodes[0]
+
+            for node in callback_nodes[1:]:
+                previous_node.next_node = node
+                previous_node = node
+        return callback_nodes
+
+    def add_done_callback(self, *callback_nodes):
+        """
+        Add a callback when channel ends correctly (and without raising Dropped)
```

Comment:
language is wrong.

We always add the callback.

perhaps something shorter to find but what's correct is:

"Add a callback, that will be called, when ...."